app/templates/create_buyer/create_buyer_account.html

```diff
@@ -32,7 +32,7 @@
 {% include "toolkit/page-heading.html" %}
 {% endwith %}
 
-<p class="lede">
+<p class="govuk-body-l">
   You must be employed by, or represent, a public sector organisation.
 </p>
 
```

Comment:
Missed on at L45 of this file https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/bcba80b930d91fbf5353b50b59e3b71387d2a88e/app/templates/create_buyer/create_buyer_account.html#L45